lisk-dex-electron/src/App.jsx

```diff
@@ -605,63 +603,70 @@ class App extends React.Component {
 
   render() {
     if (!this.state.configurationLoaded) {
-      return <div style={{ padding: '10px' }}>Loading...</div>
+      return <div style={{ padding: '10px' }}>Loading...</div>;
     }
-    return <>
-      <userContext.Provider value={{ ...this.state }}>
-        {this.state.displaySigninModal && <SignInModal failure={this.state.signInFailure} passphraseSubmit={this.passphraseSubmit} enabledAssets={this.state.enabledAssets} close={this.closeSignInModal} walletGenerated={this.walletGenerated}></SignInModal>}
-        {this.state.displayLeaveWarning && <LeaveWarning setDisplayLeaveWarning={this.setDisplayLeaveWarning}></LeaveWarning>}
-        <div className="top-bar">
-          <div>
-            <b style={{ fontSize: '21px' }}>{this.state.configuration.appTitle}</b> &nbsp;
-            <a className="feedback-link" style={{ color: '#34cfeb', fontSize: '14px' }} href={this.state.configuration.feedbackLink.url} rel="noopener noreferrer" target="_blank">{this.state.configuration.feedbackLink.text}</a>
-          </div>
-          <div>
-            <SignInState className="sign-in-state" showSignIn={this.showSignIn} keys={this.state.keys} signedIn={this.state.signedIn} signOut={this.signOut}></SignInState>
-          </div>
-        </div>
-        <div className="container">
-          <div className="notifications">
-            {this.state.notifications.map(data => <Notification key={data.id} data={data}></Notification>)}
-          </div>
-          <div className="sell-panel">
-            <PlaceOrder side="ask" orderSubmit={this.orderSubmit} orderSubmitError={this.orderSubmitError}></PlaceOrder>
-          </div>
-          <div className="buy-panel">
-            <PlaceOrder side="bid" orderSubmit={this.orderSubmit} orderSubmitError={this.orderSubmitError}></PlaceOrder>
+    return (
+      <>
+        <userContext.Provider value={{ ...this.state }}>
+          {this.state.displaySigninModal && <SignInModal failure={this.state.signInFailure} passphraseSubmit={this.passphraseSubmit} enabledAssets={this.state.enabledAssets} close={this.closeSignInModal} walletGenerated={this.walletGenerated} />}
+          {this.state.displayLeaveWarning && <LeaveWarning setDisplayLeaveWarning={this.setDisplayLeaveWarning} />}
+          <div className="top-bar">
+            <div>
+              <b style={{ fontSize: '21px' }}>{this.state.configuration.appTitle}</b>
+              {' '}
```

Comment:
What is this? Looks strange.

Reply:
Do we really need a space after apptitle, we can remove it if we don't need it :)